mteb/tasks/Retrieval/fr/BSARDRetrieval.py

```diff
@@ -12,13 +12,13 @@ class BSARDRetrieval(AbsTaskRetrieval):
         name="BSARDRetrieval",
         description="The Belgian Statutory Article Retrieval Dataset (BSARD) is a French native dataset for studying legal information retrieval. BSARD consists of more than 22,600 statutory articles from Belgian law and about 1,100 legal questions posed by Belgian citizens and labeled by experienced jurists with relevant articles from the corpus.",
         reference="https://huggingface.co/datasets/maastrichtlawtech/bsard",
-        hf_hub_name="mteb/bsard",
+        hf_hub_name="maastrichtlawtech/bsard",
         type="Retrieval",
         category="s2p",
         eval_splits=["test"],
         eval_langs=["fr"],
         main_score="ndcg_at_100",
-        revision="5effa1b9b5fa3b0f9e12523e6e43e5f86a6e6d59",
+        revision="5e1b70b3be3f5bd4676d3cd59bd01fb17f3ce722",
```

Comment:
We ran evals on this version: `5effa1b9b5fa3b0f9e12523e6e43e5f86a6e6d59`. We should keep it for consistency, no?
```suggestion
        revision="5effa1b9b5fa3b0f9e12523e6e43e5f86a6e6d59",
```